### PR TITLE
Merge changes from alpha into master branch

### PR DESCRIPTION
**Pull request by hilott**
Also update README for v2.0.0 https://github.com/jubilee-works/timetree-sdk-js/pull/275/commits/5d06a23c6099ce3ea6c143c4c7a45b157bae307d

### CODE REVIEW

File: web-api/README.md [hunk 1]
```diff
@@ -16,13 +16,13 @@ $ yarn add @timetreeapp/web-api
 ```
 ## Usage
 
-### Request to TimeTree Web API
-https://codesandbox.io/embed/flamboyant-haze-jfi63?fontsize=14&hidenavigation=1&theme=dark
+### Accessing API endpoints as a User App
```
Comment (fujikky) [inline web-api/README.md:19]: `User App` has been renamed to `OAuth App`

```suggestion
### Accessing API endpoints as a OAuth App
```

File: web-api/README.md [hunk 2]
```diff
@@ -148,6 +148,28 @@ console.log(response.accessToken);
...
 
+### Accessing API endpoints as a Calendar App
+
+```ts
+import { OAuthAuthenticator, CalendarAppClient } from "@timetreeapp/web-api";
+
+// Generate and download the private key. Please refer to the link below.
+// https://developers.timetreeapp.com/docs/api/calendar-app
+const authenticator = new CalendarAppAuthenticator({
+  applicationId: "<your-jwt-application-id>",
```
Comment (fujikky) [inline web-api/README.md:159]: The documentation seems to name it `CALENDAR_APP_ID`
https://developers.timetreeapp.com/en/docs/api/calendar-app#when-a-user-unlinks-your-calendar-app

```suggestion
  applicationId: "<your-calendar-app-id>",
```

**Review by fujikky — APPROVED** [review]
LGTM 👍  (nits commented)

**Comment by hilott** [comment]
@fujikky Thank you for suggestions! Updated.

**Pull request merged**